tests/unit/cfg/aws_services_config

```diff
@@ -0,0 +1,10 @@
+[default]
+endpoint_url = https://localhost:1234/
+services = my-services
+
+[services my-services]
+s3 =
+  endpoint_url = https://localhost:5678/
+  addressing_style = path
+dynamodb =
```

Comment:
Should we add a test case for one of the services in the list where its being renamed?

Reply:
The functional tests (both the one I modified in `test_endpoints.py` and the ones for the config provider) enumerates all available services, which should address this?

Reply:
I just think it could be beneficial to have one in here too. Not necessary though.